benefits/settings.py

```diff
@@ -22,7 +22,7 @@ def _filter_empty(ls):
 
 ADMIN = os.environ.get("DJANGO_ADMIN", "False").lower() == "true"
```

Comment:
I don't see any other references to this setting, so I guess we should delete it too?

Reply:
My suggestion was to do the minimal amount of change to get the test passing. The full removal of `DJANGO_ADMIN`  would be done in #1856